src/Pages/ProductForm.tsx

```diff
@@ -120,7 +122,7 @@ export const ProductForm = () => {
 
   const onSubmit = async (data: FormValues) => {
     await axios
-      .post("http://localhost:4000/create-product", data) // {} => to jest body ( tu dodac body frontend)
+      .post("http://localhost:4000/create-product", data, {})
```

Comment:
po co dawac pusty obiekt jako `options`?
```suggestion
      .post("http://localhost:4000/create-product", data)
```